pkgs/development/python-modules/rfeed/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python-dateutil
+, lxml
+}:
+
+buildPythonPackage rec {
+  pname = "rfeed";
+  version = "1.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-qpUG8oZrdPWjItOUoUpjwZpoJcLZR1X/GdRt0eJDSBk=";
+  };
+
+  meta = with lib; {
+    description = "Extensible RSS 2.0 Feed Generator written in Python.";
+    downloadPage = "https://github.com/svpino/rfeed/releases";
+    homepage = "https://github.com/svpino/rfeed";
+    license = with licenses; [ mit ];
```

```suggestion
    license = lib.licenses.mit;
```
for only one license, it's ok to set it directly.